src/SeleniumLibrary/keywords/formelement.py

```diff
@@ -421,8 +422,15 @@ def _get_value_from_radio_buttons(self, elements):
                 return element.get_attribute('value')
         return None
 
-    def _input_text_into_text_field(self, locator, text, clear):
+    def _input_text_into_text_field(self, locator, text, clear, disable_log):
```

Comment:
`clear` and `disable_log` should have default values. They provide kind of type hints and at least the latter wouldn't need to be always passed.